openmc/tallies.py

```diff
@@ -1996,6 +2001,10 @@ def _align_tally_data(self, other, filter_product, nuclide_product,
             other._std_dev = np.repeat(other.std_dev, filter_copy.num_bins, axis=0)
             other.filters.append(filter_copy)
 
+        # Create a new static copy of the other tally to use in swapping filters
+        if len(other_missing_filters) > 0:
```

Comment:
Isn't `other_old` already used as a variable name, since it is the name of one of the method params?